packages/lit-dev-content/site/tutorials/content/working-with-lists/01.md

```diff
@@ -0,0 +1,66 @@
+While Lit allows rendering values straight out of an iterable, it would be nice
+to have the ability to apply some transformation like adding markup for each
+item. Lit provides a built-in `map` directive that works on any iterable such as
+arrays, sets, maps, and even generators. It returns an iterable containing the
+result of calling the provided callback function on each item.
+
+In this example, a custom element is presented with a property named `items`
+that contains a set of strings.
+
+Render each item in the set as a list item (`<li>`) containing that string.
+Begin by importing the `map` directive.
+
+{% switchable-sample %}
+
+```ts
+// my-element.ts
+import {map} from 'lit/directives/map.js';
+```
+
+```js
+// my-element.js
+import {map} from 'lit/directives/map.js';
+```
+
+{% endswitchable-sample %}
+
+Use the directive in the rendered template, providing a callback function that
```

Comment:
I don't know about "rendered template" - all templates are ultimately rendered, yeah? So just "template" or maybe "component's template" if you want something more?